src/app/containers/Ad/Amp/AdSlot.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { oneOf, string } from 'prop-types';
+import {
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+} from '@bbc/gel-foundations/breakpoints';
+
+const constructAdJsonData = ({ service }) => ({
+  targeting: {
+    slot: 'leaderboard',
+    asset_type: 'index',
+    channel: service,
+  },
+});
+
+const slotConfigurations = {
+  leaderboard: {
+    mobile: {
+      width: '320',
+      height: '50',
+      media: `(max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX})`,
+      adMultiSize: '320x50,300x50',
+    },
+    desktop: {
+      width: '970',
+      height: '250',
+      media: `(min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN})`,
+      adMultiSize: '970x250,728x90',
+    },
+  },
+  mpu: {
+    mobile: {
```

Comment:
You'd only actually one of these since it's the same between mobile/desktop. I did consider `default` & `desktop` instead of `mobile` & `desktop`. In this case there would just be one default. Thoughts?

Reply:
Since the content of these are identical, can we have just a single key, 'default'?

Reply:
Should be resolved now. I opted to make "mobile" the default rather than call it default directly. It seemed less confusing as I went over it in my head